packages/common/src/items/create-item-from-url-or-file.ts

```diff
@@ -22,16 +23,28 @@ interface ICreateItemFromUrlOrFileOptions extends IUserRequestOptions {
  *
  * @export
  * @param {ICreateItemFromUrlOrFileOptions} createItemFromUrlOrFileOptions Input params (item, groups?, requestoptions)
- * @return {*}  {Promise<string>} AGO item id
+ * @return {*}  {
+ *     title: string,
+ *     createdItem: ICreateItemResponse,
+ *     itemAccessResponse: ISharingResponse,
+ *     itemSharingResponse: ISharingResponse[]
+ *   } Responses from createdItem, changing item access, and item Sharing to group
  */
 export async function createItemFromUrlOrFile(
   createItemFromUrlOrFileOptions: ICreateItemFromUrlOrFileOptions
-): Promise<ICreateItemResponse> {
+): Promise<{
+  title: string;
+  createdItem: ICreateItemResponse;
+  itemAccessResponse: ISharingResponse;
+  itemSharingResponse: ISharingResponse[];
+}> {
```

Comment:
Returning all the responses is a good idea. Also I'm pretty sure tsdoc will generate the return details